README.dev-quickstart.md

```diff
@@ -0,0 +1,35 @@
+
+# hello world
+- toto je priznanie digital
+- vsetku logiku najdes rozsekanu po featurach v adresari "src/features"
+- tam su veci zgroupovane po namespacoch "src/features/NAMESPACE/"
+- a pod nimi su potom featury "src/features/NAMESPACE/FEATURE/"
+- pod featurou su vzdy subory k danej feature
+
+
+# ako zacat
+- npm run dev
+
+
+# ako otestovat ci si nic nerozbil (CYPRESS ui e2e testy)
+- npm run cy:run
+
+
+# ako otestovat ci si nic nerozbil (JEST unit testy)
+- npm run test
+
+
+# dev faq
+- nejde mi node, hadze chyby
+    skus node 16, nainstalujes cez nvm a potom by malo ist, s novsim su issues
+- ako to nainstalujem
+    git clone a npm i
```

Comment:
pouzivame yarn a nie npm (mame commitnuty yarn.lock, nie package-lock.json). Je to napisane aj v terajsom readme, tak by som to dal aj tu, nech to nemyli

Reply:
jo, upravene